RELEASE/scripts/autoscend/auto_mr2018.ash

```diff
@@ -1329,3 +1323,39 @@ boolean fightClubStats()
 
 	return true;
 }
+
+static boolean[item] __poke_fam_equipment = $items[amulet coin, luck incense, muscle band, razor fang, shell bell, smoke ball];
+static item __fertilizer = $item[Pok&eacute;-Gro fertilizer];
+static item __tallGrass = $item[Packet Of Tall Grass Seeds];
```

Comment:
I feel like these are kind of a bad precedent to set. There are few enough uses of them that they wouldn't be onerous to replace here.
As they're local to the file, they're not a big issue right now but if we start allowing this in smaller files like this it will be more difficult to deny stuff like this in a file like autoscend.ash once the precedent has been set.
Entirely your call though.

Reply:
The "__" is a distinction i've been using to try to denote a "private" variable or function. They are only designed to be used within the scope of that file. Its the same convention python uses to denote privacy. Python loosely enforces it but you can work around it so its largely a sign that says "hey dont use this, if you do you are breaking encapsulation and opening yourself up to heartache when things change without warning."

That said, I think you're right that these dont have much benefit being in global scope, I'll move them to function scope.